skins/moono-lexicon/mainui.css

```diff
@@ -174,7 +174,11 @@ Special outer level classes used in this file:
 
 	cursor: sw-resize;
 
-	float: left;
+	/* `liferay-r2` takes care of processing external/internal CSS resources
+	on DXP and already deals with converting float directions.
+	So, we will be forcing those styles to be right because `liferay-r2`
+	will take care of converting this direction to left */
+	float: right;
```

Comment:
I tested this and it looks like it's not only `float` that gets flipped. Most other styles in `cke_resizer_rtl` do as well, for example `margin-left: -4px;` becomes `margin-right: -4px;`. This is why 'after' in this PR still looks a bit off, triangle should point to bottom left. Interestingly, `cursor` does not get flipped, I guess that's something we can improve in DXP script.

Would it worked if we removed all `cke_resizer_rtl` styles, and merged all `cke_resizer_ltr` styles into `cke_resizer`? That way we would 'hardcode' all `ltr` styles.

Reply:
> Would it worked if we removed all cke_resizer_rtl styles, and merged all cke_resizer_ltr styles into cke_resizer? 

Yes, it worked. I think it's safer going this way. The only issue I noticed was about the cursor and now I'm using *_ltr and *_rtl classes for adjusting cursor values.

Reply:
See 
7ab82dc